options.go

```diff
@@ -144,6 +144,9 @@ type Options struct {
 
 	// Disable set-lib on connect. Default is false.
 	DisableIndentity bool
+
+	// Hooks are initial hooks in options like AddHook.
+	Hooks []Hook
```

Comment:
The Hooks array should be validated during initialization to ensure that invalid or nil hooks are not added. This can help prevent runtime issues and maintain the integrity of the client setup.